src/generators/typescript/presets/DescriptionPreset.ts

```diff
@@ -37,7 +37,7 @@ export const TS_DESCRIPTION_PRESET: TypeScriptPreset = {
     self({ renderer, model, content }) {
       return renderDescription({ renderer, content, item: model });
     },
-    property({ renderer, property, content }) {
```

Comment:
We need to make it slightly more advanced, and use the options argument to conditionally write docs for properties or getters depending on whether the option for using getters is sat 🙂

Reply:
> Sorry for the late reply @akkshitgupta 🙇

I understand that you are occupied with your work as well. Thank you for being helpful as it encourages me to accept more challenges and continue my journey with AsyncAPI.


> does not matter if there are duplicate information for now.
I would like to solve it out rather than just allowing the duplicates if it not much necessary to include description for getters and hold up the PR for now. LMK WDYT

Reply:
I am getting so confused with all these PRs targeting different languages where some use getters and setters automatically and others have options to turn them on and off..

For TypeScript it's always on and can't be turned off... So your implementation is excellent 😄